cmd/install/install.go

```diff
@@ -229,3 +250,39 @@ func downloadArgoCD(osPlatform string) error {
 	}
 	return nil
 }
+
+func installCAPI(ver string, infrastructureProviders, bootstrapProviders []string) error {
```

Comment:
Can you rename ver to something more expressive like coreProviderVersion

Reply:
done